docs/overrides/home.html

```diff
@@ -244,6 +244,24 @@ <h2>TruLens: Don't just vibe check your LLM app!</h2>
                         </p>
                     </div>
 
+                <div style="margin-top: 40px;" class="logos-section">
+                    <h2>Trusted by</h2>
+                    <div class="logos-container" style="display: flex; justify-content: space-around; align-items: center;">
```

Comment:
@sfc-gh-jreini I recommend also adding the following properties:
```
flex-wrap: wrap; // This makes it wrap to the second row when we get to more logos
gap: 16px; // Optional: this just requires at least little bit of spacing between the icons
```

generally we can test this by adding more icons and playing around with resizing the browser to see if it's to our linking